src/lib/services/CommandLineSupport.ts

```diff
@@ -79,28 +79,37 @@ export abstract class CommandLineSupport extends AsyncCreatable {
 
 			// When data is passed back up to us, pop it onto the appropriate string.
 			cp.stdout.on('data', data => {
-				this.outputProcessor.processRealtimeOutput(String(data));
-				stdout += data;
+				// eslint-disable-next-line @typescript-eslint/no-floating-promises
+				(async () => {
```

Comment:
It's bad practice to make the callback itself async. Apparently the enclosed IIFE is a preferred way of indicating that the promise is intentionally left dangling.
However, that just moves the lint error one level up, so I added the next-line-disable for it.